### PR TITLE
application style improvemetns

### PR DESCRIPTION
**Pull request by vreshch**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/91" target="_blank">ReviewNinja

File: env/webpack.common.js [hunk 1]
```diff
@@ -74,6 +74,11 @@ module.exports = {
             to: 'assets',
             flatten: true
         }]),
+        new CopyWebpackPlugin([{
+            from: '**/*.svg',
```
Comment (disist) [inline env/webpack.common.js]: I guess we can extend exist pattern `'**/*.svg'` to `**/*.*(png|svg)`
instead of create new CopyWebpackPlugin again

**Comment by disist** [comment]
It will be better to fix build, please fix test `should return main page route`

File: env/webpack.common.js (hunk 1 above)
Comment (vreshch) [reply env/webpack.common.js]: - will be done

**Pull request merged**